Makefile

```
@@ -102,13 +106,19 @@ manifests/operators: operator-source
 manifests: manifests/00-catalog.yaml manifests/operators
 
 .PHONY: operator-source
-operator-source: 
-	$(call checkout_operator,openshift,dedicated-admin-operator)
+operator-source:
+	for operator in $(OPERATORS); do \
+		org="$$(echo $$operator | cut -d / -f 1)" ; \
+		reponame="$$(echo $$operator | cut -d / -f 2-)" ; \
+		echo "org = $$org reponame = $$reponame" ; \
+		$(call checkout_operator,$$org,$$reponame) ;\
+		echo ;\
+	done
 
 .PHONY: catalog
 catalog: operator-source
 	for DIR in $(SOURCE_DIR)/**/; do \
-		eval $$($(MAKE) -C $$DIR env --no-print-directory); \
+		eval $$($(MAKE) ALLOW_DIRTY_CHECKOUT=$(ALLOW_DIRTY_CHECKOUT) -C $$DIR env --no-print-directory); \
```

What's the value in making this and not all the vars used explicit?

```suggestion
		eval $$($(MAKE) -C $$DIR env --no-print-directory); \
```

---

What other variables are relevant to `make env`?

---

I checked locally and this (and other) vars seem to be inherited so this isn't explicitly needed.